### DISCUSSION TITLE
Merge storybook with a 3rd party storybook site

### DISCUSSION THREAD

**Discussion by rexkenley** [storybookjs/storybook#23581] [Help] [answered]
Is there a way to merge the storybook content from a 3rd party site to our storybook?

We want to add the components list section of https://mgt.dev/ to the components section of our storybook.

**Comment by shilman** [comment, marked as answer]
Yes, Storybook supports a feature called composition for exactly this use case: https://storybook.js.org/docs/7.1/react/sharing/storybook-composition/

**Reply by rexkenley** [reply]
Thanks! My team LOVES Storybook

**Comment by rexkenley** [comment]
@shilman 

I added the necessary configuration but I am getting an error. When I click on "Something went wrong" the mgt.dev loads fine.
![image](https://github.com/storybookjs/storybook/assets/10548251/ce38f46f-d1ac-48f8-bf1b-ed749332ff18)
![image](https://github.com/storybookjs/storybook/assets/10548251/78dfe79a-0c70-424f-9939-c402f5a4e8ad)
![image](https://github.com/storybookjs/storybook/assets/10548251/4bd26581-8087-46c6-9855-ea82dbf4fa23)

**Reply by rexkenley** [reply]
@shilman , any update?

**Comment by nathansom** [comment]
@shilman Is this feature (external stories) limited by the external site's CORS policy?
